### DISCUSSION TITLE
The dates displayed on helm repo are not good

### DISCUSSION THREAD

**Discussion by rleffd** [artifacthub/hub#4131] [Q&A] [answered]
Hello,

I have a problem with the dates that are displayed on helm repos. All versions of the charts are set to the creation date in artifacthub repo.

<img width="186" alt="image" src="https://github.com/user-attachments/assets/52c4e5c2-890a-4dbb-8d99-8b5dfc0c0963">

Version and dates:

- 1.0.0 -> 26/10/2024
- 1.0.1 -> 29/10/2024
- 1.0.2 -> 29/10/2024 

Hub Version used: 1.20.0

Is there a config I missed somewhere plz ?

**Comment by rleffd** [comment]
ok I've found on another existing instance on-prem. The dates are set when the pull action is done by the tracker. New released will appear with the correct date in future.

**Comment by tegioz** [comment, marked as answer]
Our first option is to pick the date for each chart version from the charts repository `index.yaml` file. Only when we can't find it there, we use the date when it was indexed by AH.

**Reply by tegioz** [reply]
As in: https://github.com/artifacthub/helm-charts/blob/8b3b89d9a4329bd3690729b88729ab7c1da9d475/index.yaml#L74
